js/views/modals/purchase/Purchase.js

```diff
@@ -116,24 +115,46 @@ export default class extends BaseModal {
     this.listenTo(this.coupons, 'changeCoupons',
       (hashes, codes) => this.changeCoupons(hashes, codes));
 
+    const currencies = this.listing.get('metadata').get('acceptedCurrencies') || [];
+    const disabledCurs = currencies.filter(c => !isSupportedWalletCur(c));
+    this.cryptoCurSelector = this.createChild(CryptoCurSelector, {
+      disabledMsg: app.polyglot.t('purchase.cryptoCurrencyInvalid'),
+      initialState: {
+        controlType: 'radio',
+        currencies,
+        // Kludge to set it to the first alphabetically, may not always match view sorting
```

Comment:
I'm not actually sure of the best way to handle this, the goal is to set the initial active currency to the first one in the selector, but sometimes the order is different after the selector's set state alphabetizes things.

In this example, sometimes the currencies were ['LTC', 'BTC'], and just setting it to the first one resulted in LTC being selected, which was the second after it was alphabetized.

For now I think this kludge is ok, the ideal would be to not set activeCurs, and have the crytoCurSelector automatically select the first one after alphabetizing if no value was passed in and the controlType is radio.